aspnetcore/signalr/version-differences.md

```diff
@@ -24,12 +34,12 @@ ASP.NET Core SignalR supports JSON, as well as a new binary protocol based on [M
 
 ## Differences on the server
 
-The SignalR server-side libraries are included in the `Microsoft.AspNetCore.App` package that is part of the **ASP.NET Core Web Application** template for both Razor and MVC projects.
+The ASP.NET Core SignalR server-side libraries are included in the `Microsoft.AspNetCore.App` package that is part of the **ASP.NET Core Web Application** template for both Razor and MVC projects.
```

Comment:
If this is the first mention of the package, link it like this ...

> [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app)

Contraction ... "that is" ... "that's"